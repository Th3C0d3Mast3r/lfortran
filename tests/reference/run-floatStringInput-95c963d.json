{
    "basename": "run-floatStringInput-95c963d",
    "cmd": "lfortran --no-color {infile}",
    "infile": "tests/errors/floatStringInput.f90",
    "infile_hash": "b9ed5f4fbd4e28d48deb48b01b4aea705c9759488c0a6c7c68886d88",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "run-floatStringInput-95c963d.stdout",
    "stdout_hash": "e30ae761554c4a5a9d0ee2af5cac1e16b8abbc0c6611e3fdefd8588b",
    "stderr": "run-floatStringInput-95c963d.stderr",
    "stderr_hash": "6159cdae419f2b4f5e4f98f6d62d37d848a06b93078bffa2d2bbed2e",
    "returncode": 1
}