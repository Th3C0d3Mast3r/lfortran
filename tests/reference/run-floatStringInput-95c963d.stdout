Runtime error: File `invalidInput_float.txt` does not exists!
Cannot open a file with the `status=old`
